azure-pipelines.yml

```diff
@@ -68,6 +68,10 @@ jobs:
 
       - linux32: py38-32bit
         name: py38_test_32bit
+        libraries:
+          apt:
+            - libopenjp2-7
+            - openssl-devel
```

Comment:
-devel are sufix for fedora like distros, on ubuntu its -dev

try `libssl-dev` instead and maybe `libopenjp2-7-dev` if we use lib for some compile 

-dev packages contains header files required for compilation, without dev, they are just compiled libraries.